packages/composite-checkout-wpcom/src/components/wp-order-review-line-items.js

```diff
@@ -5,7 +5,7 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 import { renderDisplayValueMarkdown } from '@automattic/composite-checkout';
-import { useTranslate } from 'i18n-calypso';
+import { useTranslate, sprintf } from 'i18n-calypso';
```

Comment:
Will `sprintf` work here? I couldn't get past the billing step to see if it blew up on me.

Reply:
Yeah, you're right. This needs to be different now that we're in calypso. [The docs for i18n-calypso](https://github.com/Automattic/i18n-calypso#string-substitution) explain how to do this with the different module. It's similar, but we can just use the second argument of the `translate` function instead with the `args` property.

Reply:
Cool, I'll update it.

Reply:
I've updated it but can't test it anymore because the delete buttons aren't rendering.